src/components/CodeSnippet.tsx

```diff
@@ -0,0 +1,60 @@
+import { Box } from '@radix-ui/themes'
+import { useEffect, useMemo } from 'react'
+
+import { ReadOnlyEditor } from './Monaco/ReadOnlyEditor'
+
+export function CodeSnippet({
+  value,
+  language = 'json',
+}: {
+  value: string
+  language: string
+}) {
+  const editorHeight = useMemo(() => {
+    const lineHeight = 20
+    const lines = value.split('\n').length
+
+    return lineHeight * lines
+  }, [value])
+
+  // Disable monaco editor focus on mount
+  useEffect(() => {
+    requestAnimationFrame(() => {
+      requestAnimationFrame(() => {
+        if (document.activeElement instanceof HTMLElement) {
+          document.activeElement.blur()
+        }
+      })
+    })
+  }, [])
```

Comment:
A workaround to disable autofocus when monaco editor is mounted, this does the trick, but if you know a better way - please let me know.